cms/migrate/05-prohibit-newline-in-module-objective.js

```diff
@@ -3,7 +3,7 @@ module.exports = function(migration) {
 
   trainingModule.editField('objective', {
     validations: [
-      { prohibitRegexp: '\n' }
+      { 'prohibitRegexp': { 'pattern': '\\n'} }
```

Comment:
Please can you add the custom error message from the screenshot to improve user experience and rename the migration file changing `newline` to `markdown` to clarify intent.

Reply:
Do you mean manually adding that message or within the migration? I'm not sure if you can do the latter

Reply:
URGH looks like you're right ☹️